website/docs/reference/project-collaboration-mode.md

```diff
@@ -2,52 +2,64 @@
+**Plan**: [Enterprise](https://www.getunleash.io/pricing) | **Version**: `4.22+`
 
 :::
 
-A project's **collaboration mode** specifies **who can submit [change requests](change-requests.md)**. There are two collaboration modes:
-- [**open**](#open-collaboration-mode)
-- [**protected**](#protected-collaboration-mode)
-- [**private**](#private-collaboration-mode)
+# Overview
 
-## **Open** collaboration mode
+Project collaboration modes allow you to manage project visibility and interaction levels, ensuring better control and collaboration within your organization.
 
-Anyone can submit change requests in an **open** project, regardless of their permissions within the project and globally. This is the default collaboration mode for projects.
+The collaboration mode of a [project](projects.md) defines who within your Unleash instance can access the project. This setting also determines who can submit [change requests](change-requests.md)–a feature that provides controlled and secure updates for feature toggles.
 
-The open collaboration mode is the default in Unleash and is how all projects worked in Unleash before the introduction of collaboration modes.
+Unleash supports three collaboration modes: **open**, **protected**, and **private**. The [predefined root and project roles](rbac.md#predefined-roles), as well as [custom root roles](rbac.md#custom-root-roles) of a user, determine what projects they can access.
 
-## **Protected** collaboration mode
+## Open collaboration mode
 
-Only admins and project members can submit change requests in a **protected** project. Other users can not submit change requests.
+All users of your Unleash instance can access the project and submit change requests. This is the default collaboration mode.
 
-## **Private** collaboration mode
 
-Private collaboration mode renders the project invisible to all **viewers** who are not project members. This means that **viewers** cannot see the project in the project list, nor can they access the project's features or locate the project anywhere within Unleash.
+## Protected collaboration mode
 
-**Editors**, **admins**, and users with **custom root roles** can still see private projects.
+All users of your Unleash instance can access the project but only Admins and project Members can submit change requests.
 
-### Change requests
 
-When you change a project's collaboration mode from open to protected, users who do not have the right permissions will lose the ability to create new change requests.
+## Private collaboration mode
 
-However, existing change requests created by these users will not be deleted. Any users with open change requests will still be able to cancel the change request, but they will **not** be able to update the change request.
+Only project Members, Admins, Editors, and users with [custom root roles](rbac.md#custom-root-roles) can access the project. Only Admins and project Members can submit change requests. 
```

Comment:
Do we need to differentiate between seeing and accessing a project here?

When it says only some users can "access" a project, that doesn't read to me like it's necessarily hidden from other users, which is a core part of the functionality here. Maybe we should clarify that the project is also hidden from viewers?

Reply:
Thanks I agree, I'll go for view everywhere